io-pay-portal-fe/src/api/services.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  fromLeft,
+  TaskEither,
+  taskEither,
+  tryCatch,
+} from "fp-ts/lib/TaskEither";
+import * as t from "io-ts";
+import { IResponseType } from "italia-ts-commons/lib/requests";
+import { PaymentRequestsGetResponse } from "../../generated/PaymentRequestsGetResponse";
+import { apiClient } from "./client";
+import {
+  ErrorResponses,
+  toDefaultResponseErrorInternal,
+  toErrorServerResponse,
+} from "./responses";
+
+export const withApiRequestWrapper = <T>(
```

Comment:
I'm not sure we actually need this logic in the frontend

Reply:
`withApiRequestWrapper` is a method to check the status code of the api, actually I can simplify the returned `ErrorsResponse` 👍